### PR TITLE
Update Flox env

### PR DESCRIPTION
**Pull request by stealthybox**
**TLDR**
this is ready to merge and should work pretty good on a Mac and maybe even Linux
______
1st we need to remove the `log`, `cache`, and `run` folders from `.flox`.
Then we add Flox's standard .gitignore.
> (it seems that when we `flox pull --copy`, we don't write a .gitignore file)
> I'll give this feedback to the flox eng team
> I'll let the team know about ./.flox/env/* colliding with your existing gitignore for env configs.

With the `./flox` situated, we just need to update the services section to:
- activate the venv for the backend
- use the proper yarn script name

I renamed the flox env within the `env.json` to match the project.

Then you can start everything:
```bash
flox activate
ollama pull llama3
flox services start
flox services logs --follow
```

alternatively:
```bash
# in one terminal
flox activate -s
flox services logs --follow
# in a second terminal
flox activate
ollama pull llama3
```

This is tested to work on a fresh directory with my Mac.
The single log tail is quite nice for monitoring program when building a Wikipedia graph.

*note:* I did have to install the `Graph Data Science` library plugin in neo4j-desktop to get past some KNN related graph generation errors.

My WSL2 environment sadly ran out of diskspace,
so I still need to test e2e on that -- hahaha.
I did get neo4j desktop for Windows installed.

### CODE REVIEW

**Comment by stealthybox** [comment]
If you wanted to automate the pulling of the `llama3` model with a flox service, this is how:
```toml
models.command = """
# wait for ollama to be ready
until ollama list; do
  sleep 0.1
done

ollama pull llama3
"""
```
You could also put it at the end of the activate hook after the requirements.txt and `yarn install`.

This is a 4.7GB model though, so it may work nicely for people to run it themselves.
We automate it in the verba example: https://github.com/stealthybox/verba-with-flox/blob/92f512a/.flox/env/manifest.toml#L115-L123

**Comment by stealthybox** [comment]
This works on WSL2
(my device slept before I finished processing, but it did fire up llama3 and spin up the fans)

I found that this WSL2 config can work well:

`%USERPROFILE%/.wslconfig`
```ini
[wsl2]
memory = 8GB
networkMode = mirrored
```

- I needed to increase memory available to the linux VM, because the Ollama logs were showing that there was not enough memory to fit the model (5.9GiB)
- I enabled networkMode mirrored, because it's easier and more private than calculating the WSL vm gateway IP and then configuring the dbms in neo4j-desktop to listen on 0.0.0.0 instead of localhost
